### PR TITLE
[BACKLOG-15720] Searching Files in Repository

### PR DESCRIPTION
**Pull request by e-cuellar**

### CODE REVIEW

**Comment by e-cuellar** [comment]
@ddiroma Please Sir!

File: plugins/file-open-save-plugin/impl/core/src/main/javascript/app/app.module.js [hunk 1]
```diff
@@ -62,7 +62,13 @@ define([
         .component(breadcrumbComponent.name, breadcrumbComponent.options)
         .component(filesComponent.name, filesComponent.options)
         .directive(renameDirective.name, renameDirective.options)
-        .service(dataService.name, dataService.factory);
+        .service(dataService.name, dataService.factory)
+        .filter('capitalize', function() {
```
Comment (ddiroma) [inline plugins/file-open-save-plugin/impl/core/src/main/javascript/app/app.module.js]: This could be done with CSS...no need for a filter. @e-cuellar

**Comment by ddiroma** [comment]
@bmorrise 
Adding Ben for review.

**Comment by ddiroma** [comment]
I approve this PR. @bmorrise @e-cuellar

**Pull request merged**